var/spack/repos/builtin/packages/py-torchvision/package.py

```diff
@@ -102,15 +109,12 @@ class PyTorchvision(PythonPackage):
     depends_on("pil@5.3:8.2,8.4:", when="@0.13: backend=pil", type=("build", "run"))
     depends_on("py-accimage", when="backend=accimage", type=("build", "run"))
     depends_on("libpng@1.6.0:", when="backend=png")
-    depends_on("jpeg")
-
-    # Many of the datasets require additional dependencies to use.
-    # These can be installed after the fact.
+    depends_on("jpeg", when="backend=jpeg")
```

Comment:
I _think_ jpeg is only required when you want to use it as a backend, but we will find out.

Reply:
Ah, this is from https://github.com/spack/spack/pull/29614. Probably best not to rock the boat too much.